packages/editor/src/components/DataSource/DataSourceItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { HStack, Tag, Text, CloseButton } from '@chakra-ui/react';
+import { ComponentSchema } from '@sunmao-ui/core';
+import { css, cx } from '@emotion/css';
+
+const ItemStyle = css`
+  &:hover {
+    background: var(--chakra-colors-blue-50);
+  }
+`;
+const TextStyle = css`
+  &.active {
+    color: var(--chakra-colors-blue-600);
+  }
+`;
+
+interface Props {
+  dataSource: ComponentSchema;
+  tag: string;
+  name: string;
+  active?: boolean;
+  colorMap?: Record<string, string>;
+  onItemClick: (dataSource: ComponentSchema) => void;
+  onItemRemove: (dataSource: ComponentSchema) => void;
+}
+
+export const DataSourceItem: React.FC<Props> = props => {
+  const {
+    dataSource,
+    active,
+    colorMap = {},
+    tag,
+    name,
+    onItemClick,
+    onItemRemove,
+  } = props;
+
+  return (
+    <HStack padding="2" display="flex" className={cx(ItemStyle)}>
```

Comment:
`className={cx(ItemStyle)}` can be `style={ItemStyle}`. Keep the same coding style with other components.

Reply:
Do you mean change `className={cx(ItemStyle)}` to `className={ItemStyle}`? I try to change it to `style={ItemStyle}`, but it would report an error.

Reply:
Yes, you are right. It should be `className={ItemStyle}`.